src/Test/Unit/Process/Deploy/InstallUpdate/ConfigUpdate/SearchEngine/ConfigTest.php

```diff
@@ -246,9 +241,22 @@ public function testGetWithElasticSearchDataProvider(): array
                     'port' => 1234,
                 ],
                 'expected' => [
-                    'engine' => 'elasticsearch5',
-                    'elasticsearch5_server_hostname' => 'localhost',
-                    'elasticsearch5_server_port' => 1234,
+                    'engine' => 'elasticsearch6',
+                    'elasticsearch6_server_hostname' => 'localhost',
+                    'elasticsearch6_server_port' => 1234,
+                ],
+            ],
+            [
+                'customSearchConfig' => [],
+                'version' => '7.2',
```

Comment:
this dataprovider checks only > sign in version checking, you need to check and equality too

Reply:
Done